casper/src/main/scala/coop/rchain/casper/Validate.scala

```diff
@@ -141,6 +151,49 @@ object Validate {
       }
     }
 
+  /**
+    * Validate no deploy by the same (user, millisecond timestamp)
```

Comment:
I think you also need to include `term` in the uniqueness key. The same user can deploy two different pieces of code at the exact same time and that is fine.

Reply:
Look like we will rRemove term from deploy proto. 
https://rchain.atlassian.net/browse/RHOL-541

Reply:
True, but I was talking about the `term` in `DeployString`. It's fine though, we talked today in standup that we will no longer use `term` as part of the uniqueness key for other reasons, which means the PR is fine as is.